tests/models/rag/test_retrieval_rag.py

```diff
@@ -288,33 +256,6 @@ def test_custom_hf_index_retriever_save_and_from_pretrained_from_disk(self):
             out = retriever.retrieve(hidden_states, n_docs=1)
             self.assertTrue(out is not None)
 
-    def test_legacy_index_retriever_retrieve(self):
-        n_docs = 1
```

Comment:
I think we don't need to test the "legacy" case anymore after (not merged yet) #27748